app/helpers/organisation_helper.rb

```diff
@@ -125,7 +125,8 @@ def organisation_display_name_including_parental_and_child_relationships(organis
       child_relationships_link_text = child_organisations.size.to_s
       child_relationships_link_text += child_organisations.size == 1 ? " public body" : " agencies and public bodies"
 
-      organisation_name += link_to(child_relationships_link_text, organisation.public_path, class: "brand__color")
+      organisation_name += link_to(child_relationships_link_text, organisation.link_to_section_on_organisation_list_page, class: "brand__color")
```

Comment:
Hmm, yeah I guess this is a slightly philosophical point of does it belong to the organisations page or the individual organisation. Going to leave it how it is since there's not an individual model for the organisations page.